index.js

```diff
@@ -1,10 +1,8 @@
 'use babel';
 
 import JSCS from 'jscs';
-import { Range } from 'atom';
-import { findFile } from 'atom-linter';
-import { readFileSync } from 'fs';
-import stripJSONComments from 'strip-json-comments';
+import path from 'path';
+import configFile from './node_modules/jscs/lib/cli-config';
```

Comment:
@iam4x Importing the module from the JSCS package like this seems terrible. Do you know a better way?